mtp_api/apps/security/tests/test_checks.py

```diff
@@ -226,6 +248,71 @@ def test_cannot_reject_an_accepted_check(self, mocked_now):
         self.assertEqual(check.actioned_by, existing_check_user)
         self.assertNotEqual(check.actioned_at, mocked_now())
 
+    def test_can_reject_check_with_rejection_reason(self):
+        # Setup
+        users = make_test_users(clerks_per_prison=1)
+        prisoner_locations = load_random_prisoner_locations()
+        generate_payments(payment_batch=50)
+        generate_prisoner_profiles_from_prisoner_locations(prisoner_locations)
+        generate_sender_profiles_from_payments(number_of_senders=1, reassign_dcsd=True)
+        check_list = generate_checks(number_of_checks=1, create_invalid_checks=False)
+        assert check_list[0].status == 'pending'
```

Comment:
or could select on from the list with something like `next(filter(lambda check: check.status == 'pending', check_list))`